orangecontrib/text/widgets/owsentimentanalysis.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+from nltk.corpus import opinion_lexicon
+from PyQt5.QtWidgets import QApplication
```

Comment:
The widget does not really need `QApplication`. I think the preferred way is to define a function `main` at the bottom of the module and import `QApplication` locally. (See below.)